File: Python/lista2 - estruturas de repeticao/exercicio_3.py
i = 1

for i in range(10):
    name = input('Informe o nome do cliente: ')
    amount = float(input('Informe o valor da compra no ano passado: '))

    if amount < 1000:
        amount *= 0.1
        print(f'cliente: {name} valor do bônus: {amount}')
    else:
        amount *= 0.15
        print(f'cliente: {name} valor do bônus: {amount}')

    i += 1

File: C/lista2 - estrutura de repeticao/exercicio_3.c
#include <conio.h>
#include <stdio.h>

int main() {
	char name[40];
	float value, bonus;
	int i;
	
	for (i= 0; i< 10; i++) {
		printf("Informe um nome: ");
		scanf("%s", &name);
		printf("Informe o valor das compras do ano passado: ");
		scanf("%f", &value);
		
		if (value < 1000) {
			bonus = value * 0.1;
			printf("Nome do cliente: %s \n", name);
			printf("Valor do bonus: %.1f \n", bonus);
		} else {
			bonus = value * 0.15;
			printf("Nome do cliente: %s \n", name);
			printf("Valor do bonus: %.1f \n", bonus);
		}
	}
	
	getch();
	return 0;
}